lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -3117,35 +3117,31 @@ void BoundInterpreterFunction::fwdIntLookupTableInst(
 void BoundInterpreterFunction::fwdConvertToInst(const glow::ConvertToInst *I) {
   Tensor *source = getTensor(I->getInput());
   Tensor *dest = getTensor(I->getResult());
-  if (source->getType() == dest->getType()) {
+  auto srcElType = source->getType().getElementType();
+  auto destElType = dest->getType().getElementType();
+  if (srcElType == destElType) {
     // This is a noop conversion.
     dest->copyRawFrom(source);
     return;
   }
-  switch (source->getElementType()) {
-  case ElemKind::FloatTy:
-    switch (dest->getElementType()) {
-    case ElemKind::Float16Ty:
-      dest->copyWithCast<float16_t, float>(source);
-      return;
-    case ElemKind::Int64ITy:
-      dest->copyWithCast<int64_t, float>(source);
-      return;
-    default:
-      llvm_unreachable("Conversion not supported");
-    }
-    return;
-  case ElemKind::Float16Ty:
-    assert(dest->getElementType() == ElemKind::FloatTy &&
-           "Conversion not supported");
-    dest->copyWithCast<float, float16_t>(source);
-    return;
-  case ElemKind::Int64ITy:
-    assert(dest->getElementType() == ElemKind::FloatTy &&
-           "Conversion not supported");
-    dest->copyWithCast<float, int64_t>(source);
-    return;
-  default:
-    llvm_unreachable("Type not supported");
-  }
+
+#define CONVERT(_T_FROM_, _T_TO_, _DTY_FROM_, _DTY_TO_)                        \
```

Comment:
nit: typically we do not have `_` params in the macro. Could you make it. T_FROM, T_TO, etc.

Reply:
Well, all macros in OperatorTest.cpp use it for `_OP_NAME_` but not for other params. I guess that's the convention... I'll do the same then.